src/TodoBoard.js

```diff
@@ -0,0 +1,64 @@
+import {useEffect, useState} from "react";
+import axios from "axios";
+import ReadOnlyItem from "./ReadOnlyItem";
+import AddTodo from "./AddTodo";
+
+const URL = "https://jsonplaceholder.typicode.com/todos"
+export default function TodoBoard() {
+    const [todoItems, setTodoItems] = useState([]);
+
+    useEffect(()=>{
+        fetchTodosList();
+    },[])
+
+    const fetchTodosList = () => {
+        axios.get(URL)
+            .then((res)=>{
+                setTodoItems(res.data.slice(0,3));
+            })
+    }
+
+    const handleMarkCompleted = (item) =>{
+        axios.put(URL,{...item,completed:true})
+            .then((res) =>{
+                console.log(res.data)
+                // setTodoItems(
```

Comment:
Please don't keep the commented code and console logs here, also for network requests axios move them to a separate directory i.e utils/services.

Ideas are to keep the component as simple as possible and just add the component-related stuff in it and keep all the utility methods/network requests separate.